src/main/java/StringUtils.java

```diff
@@ -0,0 +1,38 @@
+import exception.StringAddCalculatorException;
+
+import java.util.Arrays;
+
+import static exception.StringAddCalculatorExceptionCode.NEGATIVE_TEXT;
+
+public class StringUtils {
+
+	public static String[] split(String delimiter, String text) {
+		if(isEmpty(delimiter)
+		   || isEmpty(text)){
+			return null;
+		}
```

Comment:
가독을 해칠 정도로 길지 않다면 되도록 한줄에 써보면 어떨까요? 
이부분도 `null`을 리턴하고 있네요 👀 

```suggestion
		if (isEmpty(delimiter) || isEmpty(text)) {
			return null;
		}
```